lib/phoenix/channel.ex

```diff
@@ -353,7 +332,22 @@ defmodule Phoenix.Channel do
           push socket, "feed", %{list: feed_items(socket)}
           {:noreply, socket}
         end
-
     """
   end
+
+  @doc """
+  Adds key/value pair to socket assigns.
+
+  ## Examples
+
+      iex> socket.assigns[:token]
+      nil
+      iex> socket = assign(socket, :token, "bar")
+      iex> socket.assigns[:token]
+      "bar"
+
+  """
+  def assign(socket = %Socket{}, key, value) do
```

Comment:
I have moved this here only so we don't need to look at two different modules for functions. Everything is in Phoenix.Channel.